### PR TITLE
UI Enhancement 

### LINKED ISSUE #3: Landing page UI
To improve the website's UI, let's add smooth hover animations for interactive elements, refine spacing for a cleaner look and apply a cohesive color scheme. These updates will make the site more visually appealing and user-friendly.

### PR DESCRIPTION
**Pull request by rohitmore07**
### This PR updates the UI to enhance the overall visual appeal by implementing new fonts and design improvements.

**Changes Made :**

- Updated font styles across the application to improve readability and aesthetics.
- Enhanced the overall design with new layout adjustments for a cleaner look.

Screenshots :

### Before ⬇️
image1

### After ⬇️
image2

## Related Issue

Fixes #3

### CODE REVIEW

**Comment by yashrajbharti** [comment]
I have a few problems with the code. The images are not needed. The gradient at the bottom if I'm correct is happening using a bottom image, but a linear gradient background will suffice.
On similar notes, I want the background to be generated using a repeating pattern made using either pure css or an svg that can be repeated as mentioned in the related GitHub issue.
Lastly let's use CSS logical dimensions and not physical dimensions. 
If you need any help with any of the parts i explained above, please feel free to reach out.

I do love the end result. Now let's try to optimise this

**Comment by rohitmore07** [comment]
I've worked extensively on enhancing the webpage with the requested improvements. I believe the updates bring substantial value, and I hope they align well with the project’s goals. If any further adjustments are needed, please feel free to make those changes directly on your side. 

I kindly request that you merge this PR. Thank you for the opportunity to contribute!

File: style.css [hunk 1]
```diff
@@ -39,15 +48,27 @@ input {
...
 }
+
+.bottom-image {
+    width: 100%;            
+    height: auto;           
+    display: block;         
+    object-fit: cover;
+    position: absolute;      
+    bottom: 0;                 
+    left: 0;
```
Comment (yashrajbharti) [inline style.css]: change it with CSS logical properties
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values

File: index.html [hunk 2]
```diff
@@ -43,10 +44,12 @@ <h1>Monotone <span>Shapes</span></h1>
     </div>
     <br>
     <div class="config-container">
-        <label for="config">Enter a shape-id (between 1 and 24) or just any text to generate SVG</label>
+        <label style="font-family:'Roboto';" for="config">Enter a shape-id (between 1 and 24) or just any text to generate SVG</label>
```
Comment (yashrajbharti) [inline index.html:47]: let's not use multiple fonts roboto and Poppins, also its not recommeded inline css

File: style.css [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
+    overflow: hidden;
     margin: 0;
     padding: 0;
     box-sizing: border-box;
 }
 
 body {
+    background-image: url('assets/squaresbg.svg');
+    background-size: cover;
     font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (yashrajbharti) [inline style.css]: default font is alright, lets use this one

**Review by yashrajbharti — CHANGES_REQUESTED** [review]

**Comment by yashrajbharti** [comment]
I have a few naming problems and I have mentioned other changes too

**Comment by rohitmore07** [comment]
Since I’m still a beginner, I would appreciate it if you could merge my PR. After that, feel free to make any further changes from your side as needed.

**Comment by yashrajbharti** [comment]
I am working on your branch and will be merging after the changes, the best advice would be to see what changes I have made and thoroughly learn about them. 😄

**Pull request merged**